spec/spec.md

```diff
@@ -516,42 +516,130 @@ The following two serviceEndpoints MUST be supported in the DID Document, but on
 
 ### Revocation
 
-StatusList2021 MUST be used for revocation of VCs, as defined in [[ref: Status List 2021]].
-
-#### credentialStatus
+The profile utilizes [[ref: Status List 2021]] specification and outlines the status information of VCs as binary values.  
+When an issuer desires to enable status information for a verifiable credential, they MAY add a status property that uses the data model described in this specification. Credential status (credentialStatus) property in the Verifiable Credential provides the information on the current status of the credential. The issued VC may include a credentialStatus property, as defined in section 2.1 of [[ref: Status List 2021]]. An Issuer of a VC must use the HTTPS URL Structure or the DID Relative URLs stored in an ID Hub to host the status list as outlined in the specification. The issuer keeps a bitstring list of all verifiable credentials it has issued. Each verifiable credential is associated with a position in the list. If the binary value of the position in the list is 1 (one), the verifiable credential is revoked, if it is 0 (zero) it is not revoked.
+
+#### DID-Relative URL Structure
```

Comment:
looking at DID-Core spec, looks like the convention is to say `relative DID URL` as opposed to `DID-Relative URL`. suggest to make a replacement throughout the PR text.